packages/client/components/EditorLinkChanger/EditorLinkChanger.tsx

```diff
@@ -125,16 +125,16 @@ const EditorLinkChanger = (props: Props) => {
           <TextBlock>
             <InputLabel>{'Text'}</InputLabel>
             <InputBlock>
-              <BasicInput {...fields.text} onChange={onChange} autoFocus name='text' />
+              <BasicInput {...fields.text!} onChange={onChange} autoFocus name='text' />
```

Comment:
Nice! I was hesitant to refactor as I figure we should probably use `useForm` instead of `withForm`, but this is a strong improvement from the non-nullable operators I was using previously